### PR TITLE
added example to powershell task.

### PR DESCRIPTION
**Pull request by dakimaz**
vtbassmatt told me how to to multiline arguments for the Powershell task. I Imagine I'm not the only one who doesn't know this, so I added an example of how to do it.

### CODE REVIEW

**Comment by PRMerger18** [comment]
@dakimaz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/tasks/utility/powershell.md [hunk 1]
```diff
@@ -129,16 +129,39 @@ Write-Host  "##vso[task.LogIssue type=error;]This is the error"
...
 >  ```ps
 > exit 1
-> ``` 
+> ```
 
 ### ApplyVersionToAssemblies.ps1
 
 [Use a script to customize your build pipeline](../../scripts/powershell.md)
 
+### Call powershell script with multiple arguments
```
Comment (vtbassmatt) [inline docs/pipelines/tasks/utility/powershell.md:142]: Please spell it "PowerShell" then I'll merge this. Thanks!

**Comment by PRMerger19** [comment]
@dakimaz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/tasks/utility/powershell.md (hunk 1 above)
Comment (dakimaz) [reply docs/pipelines/tasks/utility/powershell.md]: Done :)

**Comment by PRMerger20** [comment]
@dakimaz : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by vtbassmatt** [comment]
#sign-off

**Comment by PRMerger9** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**